contributors.md

```diff
@@ -52,3 +61,11 @@ bazel run docs
 ```
 
 You can view the documentation site at [localhost:3001](http://localhost:3001).
+
+## Deciding what to contribute
+
+A good place to start is by solving [Issues](https://github.com/dataform-co/dataform/issues), with the aim of understanding the code base well enough to create new features.
+
+We also regularly run hackathons, but you'll have to [ask us directly](mailto:opensource@dataform.co) if you would like to participate.
+
+Come meet us at [one of our events](https://www.google.com/search?q=dataform+events)!
```

Comment:
can you check with Josie if there is some alternative link / mailing list / something that she might like here?

Reply:
team@dataform.co should work